doc/install/install-gromacs.md

```diff
@@ -0,0 +1,33 @@
+# Install GROMACS with DeepMD
+## 1. Patch source code of GROMACS 
+Download source code of a supported gromacs version from https://manual.gromacs.org. Run the following command:
+```bash
+cd $deepmd_root/source/gmx && chmod +x patch.sh
+./patch.sh -d $gromacs_root -v $version -p
+```
+where `deepmd_root` and `gromacs_root` refer to source code directory of deepmd-kit and gromacs respectively. And `version` represents the version of gromacs, **only support 2020.2 now**. You may patch another version of gromacs but still setting `version` to `2020.2`. However, we cannot ensure that it works.
+## 2. Install C++ api of deepmd-kit and tensorflow
+The C++ api of `deepmd-kit 2.x` and `tensorflow 2.x` are required.
++ Tips: C++ api of deepmd and tensorflow could be easily installed from the deepmd-kit offline packages. But before using tensorflow, you need to manually change the protobuf package to [version 3.9.2](https://github.com/protocolbuffers/protobuf/releases?after=v3.11.2) in `$deepmd_env_dir/include/google/protobuf` (the offline package will install a version of 3.14, which will cause incompability). 
```

Comment:
@Ericwang6 You said that the required protobuf version is 3.9.2 but provide a link to version <3.11.2, is there any reason?

I cannot find any definition for the variable `deepmd_env_dir`. Plz provide one.

Reply:
@njzjz shall we downgrade protobuf in the offline package?

Reply:
No, it's a dependency of a package built by Anaconda Inc. Is there a reason to mix the offline package and building from source?

Reply:
@njzjz here is only a tip for users to quickly prepare a installed C++ api of `deepmd-kit` and `tensorflow`, since it is a really tough work to compile the C++ api of `tensorflow`

Reply:
> @Ericwang6 You said that the required protobuf version is 3.9.2 but provide a link to version <3.11.2, is there any reason?
> 
> I cannot find any definition for the variable `deepmd_env_dir`. Plz provide one.

Sure

Reply:
> @njzjz here is only a tip for users to quickly prepare a installed C++ api of `deepmd-kit` and `tensorflow`, since it is a really tough work to compile the C++ api of `tensorflow`

It should be officially recommended, though. We can directly provided a compiled GROMACS instead.

Reply:
Also, I think this section (Install C++ api...) is duplicate of other sections of the doc. We can assume one has already installed deepmd-kit, or just provide a link to the installation section.

Reply:
@amcadmus By the way, is it possible to avoid using any TensorFlow's API for our C++ API in our header? Then we will not need TensorFlow's header any more if we have already had C++ API library.

Reply:
> Also, I think this section (Install C++ api...) is duplicate of other sections of the doc. We can assume one has already installed deepmd-kit, or just provide a link to the installation section.

Sure, I prefer to remove this section.